### DISCUSSION TITLE
Welcome to PineScript Discussions!

### DISCUSSION THREAD

**Discussion by ArunKBhaskar** [ArunKBhaskar/PineScript#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aakash-code** [comment]
Hi this Is aakash, I am using your  Strategy - Relative Volume Gainers tradingview indicator 
this is nice work!  

Strategy - Relative Volume Gainers

i want to some guide on how can i change screener,     I want to display only those stock who has blue candle in screener is it possible not any other stock?  thanks

**Reply by ArunKBhaskar** [reply]
The results on the indicator and screener will be different. This because on the indicator volume and percentage change filter is applied. This is not possible with the screener as it generates wrong result. This happens because actual % change needs separate request. You need to turn off volume and percentage change filter to display the same results. Also the current code needs some corrections. I will update it soon.

**Comment by 13lACK-PERSON** [comment]
Hello I need help in adding an alert on a pinescript @ArunKBhaskar  here is the pine code https://www.tradingview.com/pine/?id=PUB;7628c72c3d994e11aaa99bc6d30e3dc9

**Comment by pradeepsinghchahal** [comment]
Dear, Arun. 
 I'm searching for a Pine Script Writer who can develop an indicator with alerts based on the order flow concept.  Please email me at pradeepsinghchahal@hotmail.com. My mobile number is +91 9911562095.  I look forward to your kind response.  Regards  Pradeep Singh Chahal

**Reply by WickWork** [reply]
> Dear, Arun. I'm searching for a Pine Script Writer who can develop an indicator with alerts based on the order flow concept. Please email me at [pradeepsinghchahal@hotmail.com](mailto:pradeepsinghchahal@hotmail.com). My mobile number is +91 9911562095. I look forward to your kind response. Regards Pradeep Singh Chahal

I'm willing to help!

**Comment by GridandGrit** [comment]
These are cool man thank you for posting these!

**Comment by SabdaPD** [comment]
Thanks
